include/seqan/arg_parse/arg_parse_parse.h

```diff
@@ -307,6 +308,31 @@ ArgumentParser::ParseResult parse(ArgumentParser & me,
         return ArgumentParser::PARSE_ERROR;
     }
 
+#if _SEQAN_VERSION_CHECK == 1
+    // do version check if not turned off by the user
+    std::string check = "DEV"; // default
+    if(isSet(me, "version-check"))
+        getOptionValue(check, me, "version-check");
+
+    if (!(check=="OFF"))
+    {
+        VersionCheck app_version(me.appVersionCheckFuture,
+                                 toCString(me._toolDoc._name),
+                                 toCString(me._toolDoc._version),
+                                 toCString(me._toolDoc._url));
+        checkForNewerVersion(app_version);
+
+        if (!(check=="APP_ONLY"))
```

Comment:
Yes, it can have different values. 
The user can decide whether he wants no notifications at all (OFF), all notifications (DEV - default) or only the app notifications and no "new seqan version" info (APP_ONLY).